Gruntfile.coffee

```diff
@@ -1306,13 +1306,14 @@ module.exports = (grunt) ->
 
 		eslint:
 			options:
-				configFile: if process.env.CI == "true" then ".eslintrc.ci.json" else ".eslintrc.json"
+				overrideConfigFile: if process.env.CI == "true" then ".eslintrc.ci.json" else ".eslintrc.json"
 				quiet: true
 			all:
 				src: [
 					"site/**/*.js"
 					"src/**/*.js"
 					"theme/**/*.js"
+					"*.js"
```

Comment:
@nschonni Wouldn't that render the 3 lines above useless?

Reply:
This just captures the root files like `test-runner.js`. 
Could als replace these with a full `**/*.js` with exclusions to match the ignores

Reply:
Got it. Thanks for the reply. :)

Reply:
Globs are always weird to parse, because sometimes they don't do what you expect, depending on what library they use.
When I was setting this up, I mostly bypassed Grunt and just ran `npx eslint "**/*.js"`

Reply:
Went and moved to the full glob and copied over the `.eslintignore` values. Looked at something more automated like https://github.com/sindresorhus/grunt-eslint/issues/119#issuecomment-343716068 but tripped up with the coffeescript and went back to a manual copy